blockbase/functions.php

```diff
@@ -152,7 +153,7 @@ function blockbase_fonts_url() {
 /**
  * Customize Global Styles
  */
-if ( class_exists( 'WP_Theme_JSON_Resolver_Gutenberg' ) ) {
+if ( class_exists( 'WP_Theme_JSON_Resolver' ) || class_exists( 'WP_Theme_JSON_Resolver_Gutenberg' ) ) {
```

Comment:
We call this class [in a few places](https://github.com/Automattic/themes/search?q=WP_Theme_JSON_Resolver_Gutenberg), so I think we need to determine which class is being called (plugin or Core) and reference that one? Otherwise when I try to use 5.8 + Gutenberg, I get:

`Fatal error: Uncaught Error: Class 'WP_Theme_JSON_Resolver_Gutenberg' not found in /Users/jeffreyong/Documents/THEMES/a8c-projects/themes/blockbase/inc/customizer/wp-customize-colors.php on line 56`

Reply:
I would rather have Blockbase require Gutenberg than try to ensure that what we does can work with-or-without.  If we have to check for the gutenberg function anyway to operate on 5.8 I suggest we just check for that one.

Reply:
Yeah not sure this is worth the effort, lets close.